src/core/index.md

```diff
@@ -1,63 +1,61 @@
 # Core
 
-DataJoint Core projects are fully open-source and are built to develop, define, manage,
-and visualize [data pipelines](./getting-started/data-pipelines). Below are the projects that
-make up the family of core open-source projects.
+DataJoint Core is a collection of open-source tools for building and operating scientific data pipelines.
 
 ## APIs
 
 <div class="grid cards" markdown>
 
--   :fontawesome-brands-python:{ .lg .middle } **DataJoint Python**
+-   :fontawesome-brands-python:{ .lg .middle } **DataJoint for Python**
 
     ---
 
-    A low-level client for managing [data pipelines](./glossary#data-pipeline).
+    A Python client for defining, operating, and querying data pipelines.
 
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](https://docs.datajoint.org/python/)
+    :octicons-arrow-right-24: [Documentation](https://docs.datajoint.org/python/) | [Source code](https://datajoint.com/datajoint/datajoint-matlab)
```

Comment:
Below updating 
1. To point to new mkdocs
2. matlab -> python. I assume by 'source code', we mean github, right? 
```suggestion
    :octicons-arrow-right-24: [Documentation](https://datajoint.com/docs/core/datajoint-python/) | [Source code](https://github.com/datajoint/datajoint-python)
```